status-im/status-react
# [fix #3913] do not logout when changing mailserver

fixes #3913 

### Summary:

- do not logout and remove previous mailserver
from peers when changing mailserver
- rename wnode mailserver
- move transport.inbox to mailserver.core
- fix all subs and db keys

#### Platforms
- Android
- iOS
- macOS
- Linux

#### Areas that maybe impacted
**Functional**
- mailservers
- fleet

### Steps to test:
- Select a mailserver in an account before upgrading
- After upgrade, same mailserver should still be selected
- Change selected mailserver, it should not logout and previous mailserver should disconnect (remove-peer success true in the logs).

- Selected mailserver should persist when restarting the app
- Adding custom mailserver should still work and be persisted accross restarts
- Changing fleet should still work

status: ready
